v2/contribute/decisions/session/0021-handshake-after-jwt-access-token.mdx

```diff
@@ -30,7 +30,7 @@ The backend-core handshake is mainly used to fetch a few config values and the l
 
 We can drop the handshake:
 - Anti-csrf mode can be defined on the backend
-- Access token blacklisting will be moved to the backend
+- Access token blacklisting will be moved to the backend. This will be passed to the verify/getSession functions as `checkBlacklist: boolean`
```

Comment:
Im not sure if i like the term "checkBlacklist" - cause it implies that the user needs to first blacklist the token which they may not have done. Instead, something like "checkDatabase" or "doStatefullVerification: true"